about-us.html

```diff
@@ -9,16 +9,13 @@
 <body>
     <header>
         <nav>
-            <ul>
-                <li><a href="home.html" class="home">Home</a></li>
-                <li><a href="products.html" class="shop">Shop</a></li>
-                <li><a href="about-us.html" class="about">About Us</a></li>
-                <li><a href="contact.html" class="contact">Contact</a></li>
-            </ul>
-
-            <div class="loginbar">
-                <li><a href="login.html" class="login">Login</a></li>
-                <li><a href="cart.html" class="cart">Cart</a></li>
+            <div class="nav-flexbox">
```

Comment:
instead of creating a div for the container continue using the `ul` as the container.

Let's also continue using the `li` as well. 

I think it would be best to revert to the structure you had before and just make `ul` the flexbox

Reply:
updated and sent push to repo